src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -21,17 +21,17 @@
 from sentry.utils.cursors import Cursor, CursorResult
 
 
-def get_use_case_id(request: Request) -> UseCaseKey:
+def get_use_case_id(request: Request) -> UseCaseID:
     """
-    Get useCase from query params and validate it against UseCaseKey enum type
+    Get useCase from query params and validate it against UseCaseID enum type
     Raise a ParseError if the use_case parameter is invalid.
     """
 
     try:
-        return UseCaseKey(request.GET.get("useCase", "release-health"))
+        return UseCaseID(request.GET.get("useCase", "sessions"))
     except ValueError:
         raise ParseError(
-            detail=f"Invalid useCase parameter. Please use one of: {[uc.value for uc in UseCaseKey]}"
+            detail=f"Invalid useCase parameter. Please use one of: {[uc.value for uc in UseCaseID]}"
         )
```

Comment:
Do you mean our API should accept the following for now
| strings it accept | enum it maps to |
|----|---|
|`"performance"` | `UseCaseID.TRANSACTIONS`|
|`"transactions"` | `UseCaseID.TRANSACTIONS`|
|`"sessions"`| `UseCaseID.SESSIONS`|
|`"release-health"`| `UseCaseID.SESSIONS`|


Once this pr is deployed, and the front end changes made, we will change the API again so that it will accept:


| strings it accept | enum it maps to |
|----|---|
|`"transactions"` | `UseCaseID.TRANSACTIONS`|
|`"sessions"`| `UseCaseID.SESSIONS`|

Reply:
Hey John,

If it was up to me I would just keep the external interface fixed ( i.e. emit and accept the strings "performance" and "release-health" . 

What you propose is much cleaner, and also harder.

If we change our endpoint to also accept "transactions" and "sessions" there is no harm done, we are still backward compatible.

At that point in time we can also change the UI to both accept and send "transactions" and "sessions".

Changing the backend to emit (in the response) "transactions" and "sessions" or stop accepting "release-health" and "performance" in its parameters breaks backward compatibility and I'm not quite sure what is our policy on that. 

**We need to take in account that we potentially have external users that are using the API (all our endpoints are considered public API) and if we do that we'll break them ( for no good reason IMO).**

Before breaking compatibility we should check if that's acceptable (not sure who we should ask).

As a compromise we could make the Backend accept "transactions" and "sessions" ( in addition to "release-health" and "performance" ) and maybe make the UI send "transactions" and "sessions" and also accept "transactions" and "sessions" in addition to "release-health" and "performance".

Reply:
I see, thanks!